.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
-   "extends": "next/core-web-vitals"
+   "extends": "next/core-web-vitals",
+   "rules": {
+      "react-hooks/exhaustive-deps": "off"
```

Comment:
I added it because I often don't want all the variables in the useEffect to be in the dependency array and I was getting annoyed with the warnings.

If this means I'm misusing the useEffects then I will remove that rule and we can work to improve it.

Reply:
removed